www/js/app.js

```diff
@@ -26,1533 +26,1559 @@
 // This uses require.js to structure javascript:
 // http://requirejs.org/docs/api.html#define
 
-define(['jquery', 'zimArchiveLoader', 'uiUtil', 'cookies','abstractFilesystemAccess','q'],
```

Comment:
@Hardik500 It's really hard to see what's changed and what hasn't in this file, because it seems as if everything has changed. This is because of the whitespace changes, I think. You need to re-do the PR so that only the actual code changes show up in the diff.